positron/webidl/Process.webidl

```diff
@@ -3,14 +3,36 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+dictionary ReleaseDictionary {
+  DOMString name;
```

Comment:
> Note that node process also has:

Indeed, and libUrl on Windows. This isn't a complete specification of the Node API for the *process* global, nor is the underlying implementation complete. It's just the minimum necessary to get Electron to work for the Hello World test app (and soon for a Basic Browser test app).

a7146fb adds commentary to that effect.